aarch64_extension.cpp

```diff
@@ -97,6 +169,20 @@ class AArch64ArchitectureExtension : public ArchitectureHook {
     return (BNLowLevelILFlagCondition) -1;
   }
 
+  uint32_t getVectorSubregister(uint32_t baseRegister, size_t elemSize,
+                                size_t index) {
+    for (auto& regPair : mRegisters) {
```

Comment:
Is this going to be a chokepoint being O(n)?

Reply:
Depends on how much we use it. I say don't make the data structures more complicated until it actually becomes a problem, unless you see an even cleaner way to do this.

Reply:
Seems reasonable, multi-conditionals makes it kind of annoying to use a hash-map here.